app/views/recipes/public_list.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1> Public recipes </h1>
+
+<% @recipes.includes(:user).each do |recipe| %>
```

Comment:
[OPTIONAL] Please consider changing the following line to avoid N + 1 problems.

`<% @recipes.includes(:user, :recipe_foods).each do |recipe| %>`